test/integration/api/geodata.js

```diff
@@ -264,6 +264,8 @@ describe('db: geodata parsing functions', () => {
       ['nope', null],
       ['170 80', null],
       ['90 190', null],
+      ['01.01 -002.002', null], // regression test for bug #1540
+      ['-00.11 -00.22', null], // regression test for bug #1540
```

Comment:
even precision can't have leading zeros, right?

```suggestion
      ['-00.11 -00.22', null], // regression test for bug #1540
      ['90 180 10 001', null], 

```